content/python/concepts/sets/terms/clear/clear.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'clear()' 
+Description: 'Removes all elements in the set.'
+Subjects: 
+  - 'Code Foundation'
+  - 'Computer Science'
+  - 'Data Science'
+  
+Tags:
+ - 'Methods'
+ - 'Sets'
+ - 'Functions'
+  
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+[The **clear()** method removes all elements in a python set and does not return any values. When using the `clear()` method, the set remains even after all elements in it are removed.]
+
+## Syntax
+
+`set.clear()`
+
+- The `clear()` method does not have any parameters.
+- The `clear()` method does not return any values.
+- The `clear()` method does not delete the set, it only removes elements inside the set.
+
+[!NOTE:] To editors: Should this description under syntax be removed? It kind of repeats the description above, but I thought this emphasize made it even clearer.
+
```

Comment:
```suggestion
```

Reply:
These points can be removed from here, since they are already covered in the introductory paragraph.

Reply:
Noted. Removed this section from the edited file.